vscode/src/extension.ts

```diff
@@ -1100,8 +1100,24 @@ function doActivateWithJDK(specifiedJDK: string | null, context: ExtensionContex
                 return selected ? Array.isArray(selected) ? selected : [selected] : undefined;
             });
             c.onRequest(UpdateConfigurationRequest.type, async (param) => {
-                await vscode.workspace.getConfiguration(param.section).update(param.key, param.value);
-                runConfigurationUpdateAll();
+                handleLog(log, "Received config update: " + param.section + "." + param.key + "=" + param.value);
+                if (vscode.workspace) {
```

Comment:
Is there a case where `vscode.workspace` could be `undefined`? If not, I'd drop the if.

Reply:
Thanks @lahodaj for pointing this out. I've removed the `if` check since it will always be `true`.

- `vscode.workspace` is a VSCode API [namespace](https://code.visualstudio.com/api/references/vscode-api#workspace).